nilearn/plotting/img_plotting.py

```diff
@@ -967,7 +967,7 @@ def plot_glass_brain(stat_map_img,
             absolute value will be used (rendering positive and negative
             values in the same manner). If set to false the sign of the
             maximum intensity will be represented with different colors.
-            See http://nilearn.github.io/auto_examples/manipulating_visualizing/plot_demo_glass_brain_extensive.html
+            See http://nilearn.github.io/auto_examples/manipulating_visualizing/plot_demo_glass_brain.html
```

Comment:
For example, you can see that it exists on CircleCI: https://circle-artifacts.com/gh/nilearn/nilearn/827/artifacts/0/home/ubuntu/nilearn/doc/_build/html/auto_examples/manipulating_visualizing/plot_demo_glass_brain_extensive.html